File: 01_basics/device.py
from pprint import pprint


device = {
    "name": "r3-L-K2",
    "vendor": "cisco",
    "model": "catalyst-2690",
    "os": "iosxe",
    "version": "12.2SE",
    "ip": "10.1.1.1",
}

print(device)

print()
pprint(device)

print()
for key, value in device.items():
    print(f"{key:>16s} : {value}")

File: m16_java/Device.java
package quokka.home;

import java.util.HashMap;
import java.util.Map;

public class Device {

    public static void main(String[] args) {

        HashMap<String, String> device = new HashMap<>();
        device.put("name", "sbx-n9kv-ao");
        device.put("vendor", "cisco");
        device.put("model", "Nexus90f00 C9300v Chassis");
        device.put("os", "nxos");
        device.put("version", "9.3(3)");
        device.put("ip", "10.1.1.1");

        System.out.println("device: " + device);
        System.out.println("device name: " + device.get("name"));

        System.out.println();
        for (Map.Entry<String, String> item : device.entrySet()) {
            System.out.println(item.getKey() + ":" + item.getValue());
        }
    }
}
